lib/commander/help_formatters/terminal/help.erb

```diff
@@ -6,10 +9,11 @@
 
     <%= Commander::HelpFormatter.indent 4, program(:description) %>
 
-  <%= $terminal.color "COMMANDS", :bold %>:
+  <%= "%s: %s" % [$terminal.color("COMMANDS", :bold), default_command.nil? ? '' : '(* default)'] %>
```

Comment:
I would skip the explicit `.nil?` and the ternary here, this is effectively the same as the more concise `('(* default)' if default_command)`